framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java

```diff
@@ -608,7 +610,13 @@ private void parseStubUnit(InputStream inputStream) {
     if (debugAnnotationFileParser) {
       stubDebug(String.format("parsing stub file %s", filename));
     }
-    stubUnit = StaticJavaParser.parseStubUnit(inputStream);
+    ParserConfiguration configuration = new ParserConfiguration();
```

Comment:
We may want to make this change everywhere we use StaticJavaParser.

Reply:
How about merging https://github.com/typetools/checker-framework/pull/4499 first?

Reply:
I've done that.  Thanks!